### PR TITLE
[MINOR] Improvement(dashboard): Support display human-readable time format for app page

### PR DESCRIPTION
**Pull request by maobaolong**
### What changes were proposed in this pull request?

Display the human-readable time format in application page.

### Why are the changes needed?

A long type timestamp really hard to get information from it.

### Does this PR introduce _any_ user-facing change?

Format the update time to human-readable.

### How was this patch tested?

image

### CODE REVIEW

**Comment by rickyma** [comment]
Good catch! Please remove useless auto-generated information in the description of this PR.

File: dashboard/src/main/webapp/src/pages/ApplicationPage.vue [hunk 1]
```diff
@@ -52,6 +52,7 @@
 <script>
 import { getApplicationInfoList, getAppTotal, getTotalForUser } from '@/api/api'
 import { onMounted, reactive } from 'vue'
+import { memFormatter, dateFormatter } from '@/utils/common'
```
Comment (xianjingfeng) [inline dashboard/src/main/webapp/src/pages/ApplicationPage.vue:55]:
```suggestion
import { dateFormatter } from '@/utils/common'
```

File: dashboard/src/main/webapp/src/pages/ApplicationPage.vue [hunk 2]
```diff
@@ -95,7 +96,7 @@ export default {
         getAppTotalPage()
       }
     })
-    return { pageData }
+    return { pageData, memFormatter, dateFormatter }
```
Comment (xianjingfeng) [inline dashboard/src/main/webapp/src/pages/ApplicationPage.vue:99]:
```suggestion
    return { pageData, dateFormatter }
```

**Comment by maobaolong** [comment]
@xianjingfeng Thanks for you suggestion commit, I've accepted all of them, PTAL.

**Review by xianjingfeng — COMMENTED** [review]
LGTM

**Comment by rickyma** [comment]
@xianjingfeng Please approve it.

**Comment by rickyma** [comment]
@maobaolong Please delete the auto-generated info like `(Please outline the changes and how this PR fixes the issue.)` in the description. Thanks.

**Comment by maobaolong** [comment]
@rickyma Done.

**Review by xianjingfeng — APPROVED** [review]

**Pull request merged**